README.md

```diff
@@ -75,6 +76,12 @@ These plugins will load and display on the right prompt. You can check the
 documentation and configuration for each specific plugin in the
 [plugins](/plugins) directory.
 
+You can pin* a plugin by appending `*` to the end of the name. Geometry will always render the plugin.
```

Comment:
I would add a note here on pinning, since when reading this document from the start you don't really get what it is. If I was a new user, I would think "wait, so plugins don't always render? Why not?"

What do you think of just prepending something like "Some plugins only render when you are in a given directory or in the presence of a given file. You can have them always render by pinning the plugin. (...)"?